runtime/src/test/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (C) 2016 Red Hat, Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence
+             http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
+             version="2.0">
+
+    <persistence-unit name="com.redhat.ipaas.runtime.db">
+        <provider>org.hibernate.ejb.HibernatePersistence</provider>
+        <class>com.redhat.ipaas.runtime.db.entity.Action</class>
+        <class>com.redhat.ipaas.runtime.db.entity.ActionInstance</class>
+        <class>com.redhat.ipaas.runtime.db.entity.ActionInstanceProperty</class>
+        <class>com.redhat.ipaas.runtime.db.entity.ActionProperty</class>
+        <class>com.redhat.ipaas.runtime.db.entity.ConfiguredConnector</class>
+        <class>com.redhat.ipaas.runtime.db.entity.ConfiguredConnectorProperty</class>
+        <class>com.redhat.ipaas.runtime.db.entity.Connector</class>
+        <class>com.redhat.ipaas.runtime.db.entity.ConnectorProperty</class>
+        <properties>
+            <property name="hibernate.connection.release_mode" value="after_transaction"/>
+            <property name="hibernate.dialect" value="org.hibernate.dialect.H2Dialect" />
+            <property name="hibernate.connection.url" value="jdbc:h2:./target/h2-ipaas"/>
+            <property name="hibernate.connection.driver_class" value="org.h2.Driver"/>
+            <property name="hibernate.connection.username" value="sa"/>
```

Comment:
@chirino has already shown how to set up a datasource for Spring Boot in Spring Boot application.yml so can we continue with that approach please?